### PR TITLE
Add stories to text component with HTML tags

### PR DESCRIPTION
**Pull request by migueliriano**
**Release Type:** *Non-Breaking Feature* 

Fixes https://x-team-internal.atlassian.net/browse/XP-2216

## Description
 > Added a new case in `Text.stories.js`  with `HTML` tags in the `content` for `Text` component.

## Checklist

**Before submitting a pull request,** please make sure the following is done:

- [x] set yourself as an assignee
- [x] set appropriate labels for a PR (`In Review` or `In Progress` depending on its status)
- [x] move respective JIRA issue to the `IN REVIEW` column
- [x] make sure your code lints (`npm run lint`)
- [x] Flow typechecks passed (`npm run flow`)
- [x] Snapshots tests passed (`npm run jest`)
- [x] check cleanup tasks (https://github.com/x-team/auto/labels/cleanup) and take a suitable small one (if exists) in a related area of the current changes
- [x] component's documentation (`.stories.js` file) is changed or added accordingly to reflect any new or updated use cases or variants usage.
- [x] if you've fixed a bug or added code that should be tested, add unit tests
- [x] if any snapshots have been changed, verify that component still works and looks as expected and update the changed snapshot
- [x] **manually tested the component** by running it in the browser and checked nothing is broken and operates as expected!

## Related PRs

branch | PR
------ | ------
XP-2216-update-build-portfolio-content| [Auto](https://github.com/x-team/auto/pull/1751)

## Steps to Test or Reproduce
*N/A*

## Impacted Areas in Application
- `Text.js` component

## Screenshots
text_html_content

### CODE REVIEW

**Comment by galileo** [comment]
@migueliriano There are some conflicts can we resolve them?

**Comment by migueliriano** [comment]
@galileo - Conflicts resolved

File: src/components/ui/Text.stories.js [hunk 1]
```diff
@@ -93,3 +93,6 @@ storiesOf('UI Components/Text', module)
   .add('pure content with no wrappers around and Required', () => (
     <Text content='Just a required text' isPureContent required />
   ))
+  .add('content with HTML tags', () => (
+    <Text content={(<p>Bellow you'll find questions based on the skills you told us about in level 1. <br /> <strong>You only need to respond to questions for skills that you want to continue working </strong></p>)} />
```
Comment (galileo) [inline src/components/ui/Text.stories.js]: Maybe more complex HTML sting would be more beneficial in this place?

Comment (migueliriano) [reply src/components/ui/Text.stories.js]: Comment addressed on this commit: 59d1f77eb59a93b788e32d6dbb7ab5a93a38ff1d

**Pull request merged**